src/azure-cli-core/azure/cli/core/style.py

```diff
@@ -72,3 +72,30 @@ def format_styled_text(styled_text):
     # Reset control sequence
     formatted_parts.append(Fore.RESET)
     return ''.join(formatted_parts)
+
+
+def highlight_command(raw_command):
+    """highlight a command to make it colored.
+
```

Comment:
Some example can be very helpful, like

https://github.com/Azure/azure-cli/blob/903ecf1c0e75cb1fde19c6be0ca22c52b35a7ef9/src/azure-cli-core/azure/cli/core/util.py#L1186-L1190

Reply:
Added more comments about the highlighting rules. The return type is of list of styled text, not easy to provide example.